### PR TITLE
Katib v1alpha2 kustomize package

### PR DESCRIPTION
**Pull request by richardsliu**
Split the katib manifests into v1alpha1 and v1alpha2, since we made a number of changes in v1alpha2.

Also removed the Istio virtual service in anticipation of #102.

This change is [Reviewable](https://reviewable.io/reviews/kubeflow/manifests/112)

### CODE REVIEW

File: katib/v1alpha2/base/katib-db-pv.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```
Comment (hougangliu) [inline katib/v1alpha2/base/katib-db-pv.yaml]: we don't need this file

Comment (richardsliu) [reply katib/v1alpha2/base/katib-db-pv.yaml]: Removed

**Pull request closed without merging**